source/view/index.js

```diff
@@ -353,6 +355,20 @@ vwf_view.satProperty = function( nodeID, propertyName, propertyValue ) {
                 updateBlocklyRamBar();
                 break;
 
+            case "UIEnabled":
```

Comment:
The tabs are already generated when the node is initialized (check `vwf_view.initializedNode`). Here, just check `propertyValue`. If it's `true`, call `addBlocklyTab( nodeID )`, otherwise call `removeBlocklyTab( nodeID )`.